.vscode/tasks.json

```diff
@@ -0,0 +1,55 @@
+// tasks.json
```

Comment:
Why do we need tasks.json?

Reply:
Don't *need*. I just use it as a quick full project linter. CTRL-SHIFT-P/"Run task" sometimes. I thought it would be a nice inclusion, but I can remove it.

Reply:
Removed.